Dockerfile

```diff
@@ -1,48 +1,23 @@
-FROM ubuntu:14.04
+FROM p4lang/behavioral-model:latest
```

Comment:
It is not best practice, you're right. However, there's a lot of churn in our Dockerimages right now, so in the short term this makes rapid iteration easiest. Once the rate of change slows down, we should tag specific versions.